smithy.yaml

```diff
@@ -5,7 +5,7 @@ runner_image: drydock-prod.workiva.net/workiva/smithy-runner-generator:203768 #
 
 script:
   - pub get --packages-dir
-  - xvfb-run -s '-screen 0 1024x768x24' pub run dart_dev test --pub-serve --web-compiler=dartdevc -p chrome -p vm
+  - xvfb-run -s '-screen 0 1024x768x24' pub run dart_dev test --pub-serve --web-compiler=dart2js -p chrome -p vm
```

Comment:
Why the change?

Reply:
I was getting some type errors from the dart dev compiler that didn't show up in analysis. As it turns out strong mode compliance != ddc compliance in a few annoying circumstances. Using `typed(any)` rather than `any` with mokito fixes the errors I was seeing however. https://github.com/dart-lang/mockito#strong-mode-compliance . So I'll be switching it back in an upcoming commit.